roles/confluent.control_center/tasks/main.yml

```
@@ -61,6 +61,15 @@
     group: "{{control_center_group}}"
     recurse: true
 
+# Archive File deployments need to create SystemD directory on Debian
+- name: Create Zookeeper Service path
+  ansible.builtin.file:
```

```suggestion
  file:
```
We should migrate to FQN, but for now I would prefer consistency.

---

Yeah, the latest docs actually state to use FQDN: https://docs.ansible.com/ansible/latest/collections/ansible/builtin/file_module.html

There are two approaches here, either we kick off a full project to change this across the entire project, or we start changing it as we go along.

To me, it would make sense to change it as we go along, as it would be less work for anyone person.

---

That could also be a working path, and I am happy with either. What does @domenicbove thinks?

---

@erikgb I can confirm that right now we don't have the resources internally to kick off a dedicated project for this, however if you wanted to go through and update them all I wouldn't be opposed. :).

---

My personal (and company) priority now is collection compatible layout (https://github.com/confluentinc/cp-ansible/pull/630) - to avoid the pesky submodules. And that will require a massive change in a lot of files.

As I said, I would be happy with introducing FQN gradually. If so, I think it should be mentioned in the development guidelines. And consider non-FQN as technical debt.

---

@erikgb I've changed it back to just file: and have added an internal task for us to review changing this across the board in the future.  Thanks again for the discussion, it makes sense to keep it consistent for the wider community for now.

---

IMO GitHub issues should be created for issues/tasks that are relevant for contributors and the community in general. I understand that Confluent also must track some internal work - based on tickets from supported customers. But I think it is more "community friendly" to consider GitHub issues as the master for issues/tasks in this project.

---

@erikgb good feedback.  It's something we've discussed internally before, but still not sure we have the correct direction here.  It's a constant challenge between what we keep internal, vs what is public.  For example, we have some proprietary features, that we may not want to have any public details on immediately, so we need to find a way to manage that properly.  I'll let our Product Management team know about your thoughts.